packages/loader/container-loader/src/container.ts

```diff
@@ -1860,7 +1872,12 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
             throw new Error("pkg should be provided in create flow!!");
         }
 
-        await this.loadContext(codeDetails, attributes, snapshot);
+        await this.instantiateContext(
+            codeDetails,
+            attributes,
+            false,
```

Comment:
This does not look correct. If you trace callers to createDetachedContext, you will see that it's called from two different places with different this._existing values

Reply:
Test succeeding points to lack of coverage - any chance you can glance where we test detached containers and add appropriate check?

Reply:
I see what you mean and I think I was confused by it.

In `main`, we call `createDetachedContext`:
- inside `rehydrateDetachedFromSnapshot` with `existing === true`
- inside `createDetached` with `existing === false`

this would propagate the `existing` state to the context via the container's `existing` property.

With this change, `createDetachedContext` creates the context with existing == false and it is only called inside `createDetached` which is correct. In `rehydrateDetachedFromSnapshot`, the context is created with existing == true.

But I do agree it looks weird and there is a regression. `rehydrateDetachedFromSnapshot` no longer checks that the code details are not `undefined`